test-end-to-end/webapp-cypress/cypress/e2e/001_003_move_content.cy.js

```diff
@@ -0,0 +1,51 @@
+describe('001_003_move_content', () => {
+ it('Start Ustad Test Server ', () => {
+  // Start Test Server
+    cy.ustadStartTestServer()
+})
+
+it('Admin user move content to folder', () => {
+ // Admin user login
+  cy.ustadClearDbAndLogin('admin','testpass',{timeout:8000})
+ // Add H5p File
+  cy.ustadAddContentToLibrary('../test-files/content/H5p_Content.h5p','Content_001')
+  cy.contains('Content_001').click()
+  cy.contains('OPEN').click({force: true})
```

Comment:
Why do we need force here? We shouldn't need it and normally shouldn't use it. If the button is disabled by the UI or not visible... it should wait